versionsArchived.json

```diff
@@ -1,4 +1,7 @@
 {
+  "v1.11.x": "https://docs.zowe.org/v1-11-x/",
+  "v1.10.x": "https://docs.zowe.org/v1-10-x/",
+  "v1.9.x": "https://docs.zowe.org/v1-9-x/",
```

Comment:
@nannanli once this PR is merged. I don't think the vuepress links will be working anymore.

We will either need to make standalone Docusaurus deploy links for every version in different branches. Or for now we can redirect to links as done in this [versionsArchived.json](https://github.com/covalentbond/docs-site/blob/zowe-docs-v2/versionsArchived.json)